doc/index.rst

```diff
@@ -0,0 +1,98 @@
+.. raw:: html
+
+   <style type="text/css">
+   .thumbnail {{
+       position: relative;
+       float: left;
+       margin: 10px;
+       width: 180px;
+       height: 200px;
+   }}
+
+   .thumbnail img {{
+       position: absolute;
+       display: inline;
+       left: 0;
+       width: 170px;
+       height: 170px;
+   }}
+
+   </style>
+
+pyRiemann: Biosignals classification with Riemannian Geometry
+=============================================================
+
```

Comment:
You'll have to update this index with qiskit specific description.

Reply:
Hi, you are absolutely right. In order to facilitate your review as there are already a lot of modifications in this one, do you want me to tackle the documentation in a follow-up PR and use a placeholder here?